src/components/Header.jsx

```diff
@@ -57,12 +57,12 @@ export default function NavigationHeader({ drawerAlwaysOpen = false }) {
           )}
           {!open && !drawerAlwaysOpen && (
             <Typography variant="h6" component="div" sx={{ flexGrow: 1 }} className="pageTitle">
-              Church
+              Holy
```

Comment:
We have to add  labels for  holy  in the api , what if language changes from english to other language supported in the system  `  example :  labels.holy `

Reply:
Yes for now we didn't implemented the internalisation so